src/components/dynamic-content-control/editor.scss

```diff
@@ -7,7 +7,6 @@
 .stk-dynamic-content-control {
 	display: flex;
 	align-items: flex-start;
-	overflow: hidden;
```

Comment:
Is this change okay with the other controls that use the dynamic content button?

Reply:
I'd like to suggest that so we won't affect existing styles that use this class name, write a new selector for the range control that does `overflow: auto` instead.